docs/UserGuide.adoc

```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103T-W12-2/main
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```

Comment:
Please update `By:` and `Since:` to remove SEEDU branding.

Reply:
Ok, I have updated